src/config/dynamic-theme-fixes.config

```diff
@@ -9154,6 +9154,39 @@ CSS
 
 ================================
 
+my.contabo.com
+
+CSS
+#mainmenu {
+    background-color: rgb(0, 0, 111) !important;
+    border-radius: 20px 0px 0px 20px !important;
+    box-shadow: -2px -2px, -2px 2px, 0px 2px, 0px -2px !important;
+}
+#mainmenu > div {
+    background-image: none !important;
+}
+#submenu {
+    background-color: #977122 !important;
+    border-radius: 20px 0px 0px 20px !important;
+    box-shadow: -2px -2px, -2px 2px, 0px 2px, 0px -2px !important;
+}
+#submenu > div {
+    background-image: none !important;
+}
+.button {
+    background-image: linear-gradient(#126382 49%, #0a4f6a 51%);
```

Comment:
Ideally we leave gradients as is. Is their anything wrong with it?

Reply:
it was an image befor, I replaced it with a gradiant.

Reply:
Just remove the background image that should be good enough.